examples/uppy-with-companion/server/index.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express')
 const bodyParser = require('body-parser')
 const session = require('express-session')
-const companion = require('../../../packages/@uppy/companion')
+const companion = require('../../../packages/@uppy/companion/src/companion')
```

Comment:
Is this duplication the only way it can be required?

Reply:
yes, without it will raise an import error that can't find `companion.js`.

Reply:
Can we instead list `@uppy/companion` as a dependency of the example package? I think it would be cleaner.

Reply:
@aduh95 thanks, added to the example package.